code_challenges.rb

```diff
@@ -10,6 +10,13 @@
 num3 = 221
 # Expected output: '221 is odd'
 
+def is_even_odd = num1, num2, num3 
```

Comment:
when we are declaring a method we don't use the assignment operator "=" for the values of the parameter. Parameters are dynamic placeholders that will get overwritten by the argument in the method call.